specs/README.md

```diff
@@ -1,84 +1,164 @@
 # Project Reunion Specs
 
-This directory contains archived and in-progress spec documents for APIs in
+This directory contains archived and in-progress spec documents for APIs in Project Reunion.
+
+Documents in this directory are used as part of a feature or API review for features in development
+and form the basis for public documentation and sample code. Write specs so they can be directly
+consumed by an app developer or document author.
+
+**DO** Provide feedback by **commenting on active pull requests** tagged with
```

Comment:
Nit: Capitalization after **DO** and **DO NOT** differs here than in other locations in the document.